### DISCUSSION TITLE
Virtual Meetup on Discord

### DISCUSSION THREAD

**Discussion by naveen521kk** [msys2/MINGW-packages#8335] [General]
We are planning to have a Virtual meetup on [Discord](https://discord.gg/jPQdRdDcT9), which will be held this month on a specific date that everybody comes up. This discussion is to plan things out.

When2meet: https://www.when2meet.com/?11591331-ySlfG
Agenda Etherpad (add your stuff): https://pad.ccc-mannheim.de/p/63tSa4McuD

/CC @lazka @mingwandroid

**Comment by lazka** [comment]
**5 pm utc today  /  2021-04-11T17:00:00Z**

**Comment by 1480c1** [comment]
It might be better to extend the when2meet thing to maybe about a month, so we can see with extended dates

**Comment by mingwandroid** [comment]
Thanks for attending guys. It was really nice to chat to you all!

**Comment by naveen521kk** [comment]
We are planning for another one this Saturday at 5 pm UTC. **2021-04-17T17:00:00Z**. Interested people can join on [Discord](https://discord.gg/jPQdRdDcT9).
